src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
+
+  if (century) {
+    men = men.filter(person => Math.ceil(person.died / 100) === century);
+  }
+
+  const menAge = men.map(person => person.died - person.born)
+    .reduce(reducing, 0) / men.length;
```

Comment:
нуль наче не треба писати, якшо немає кастомного першого значення
```suggestion
    .reduce(reducing) / men.length;
```